### DISCUSSION TITLE
Change background color of android bottom navigation bar.

### DISCUSSION THREAD

**Discussion by waheedakhtar694** [software-mansion/react-native-screens#1244] [Q&A] [answered]
As react-native-navigation offers options to change the background color of the Android navigation bar, is there a way in react-native-screens too like below to change the background color on the fly?
https://wix.github.io/react-native-navigation/api/options-navigationBar/
![navigationBar](https://user-images.githubusercontent.com/45425486/148029196-290c6a05-3727-4238-bb41-bc634a78a887.png)

@WoLewicki Any suggestion on this?

**Comment by WoLewicki** [comment]
It is currently not available in our library, but for sure it could be. We are always open for PRs with such functionalities and it shouldn't be too hard to implement I think. You are right that `react-native-navigation` already does it and I believe the code for these functions is in this file: https://github.com/wix/react-native-navigation/blob/9bb70d81700692141a2c505c081c2d86c7f9c66e/lib/android/app/src/main/java/com/reactnativenavigation/utils/SystemUiUtils.kt

**Reply by WoLewicki** [reply]
@waheedakhtar694 can you check if https://github.com/software-mansion/react-native-screens/pull/1264 is what you need?

**Reply by waheed25** [reply]
@WoLewicki  yes, this is what i was talking about, any idea when it will be merged?

**Reply by waheedakhtar694** [reply]
@WoLewicki I saw this PR in detail and seems like there isn't any implementation for changing the background color of the Navigation Bar on the fly like how I can make changes to the status bar, Can I with this PR?

**Reply by WoLewicki** [reply, marked as answer]
@waheedakhtar694 using `navigationBarColor` prop should change the color of navigation bar, similarly to how it is done with status bar. You can see it done e.g. here: https://github.com/software-mansion/react-native-screens/pull/1264/files#diff-676b7c94b457068affed59b63df75348e5a353566666f1939aed71230fbdaaf5R173. You can try and run the `TestsExample` project and use `Test860.tsx` in `App.js` to check all these options.
